src/endpoints/transactions.rs

```diff
@@ -104,52 +101,15 @@ pub enum DeclineReason {
     Other,
 }
 
-/// The set of categories by which Monzo transactions and merchants can be
-/// categorised
-#[derive(Deserialize, Debug, Clone, Copy)]
-#[non_exhaustive]
-#[serde(rename_all = "snake_case")]
-pub enum Category {
-    /// General expenses
-    General,
-
-    /// Restaurants, Cafes, etc
-    EatingOut,
-
-    /// Work-related expenses
-    Expenses,
-
-    /// Getting around
-    Transport,
-
-    /// Cash withdrawals
-    Cash,
-
-    /// Bills and regular expenses
-    Bills,
-
-    /// Fun and Entertainment
-    Entertainment,
-
-    /// Treat yourself
-    Shopping,
-
-    /// Holiday expenses
-    Holidays,
-
-    /// Food and household items
-    Groceries,
-}
-
 /// Merchant information which might be returned in transactions data.
 ///
 /// An id or a struct may be returned depending on whether the 'expand merchant'
 /// flag is set in the transactions request.
 #[derive(Deserialize, Debug)]
 #[serde(untagged)]
 pub enum MerchantInfo {
-    /// A unique ID associated with a merchant
-    Id(String),
+    /// A unique ID associated with a merchant (or null)
+    Id(Option<String>),
```

Comment:
semantically, if the merchant field isn't populated at all, this isn't a `MerchantInfo::Id`.

There are two options here.

1. the MerchantInfo gets a third variant which is `None`
2. the `merchant` field of the transaction becomes an `Option<MerchantInfo>`

probably the second option makes the most sense